java/src/main/java/api/example/java/model/FieldResult.java

```diff
@@ -0,0 +1,16 @@
+package api.example.java.model;
+
+import java.util.List;
+
+public class FieldResult {
```

Comment:
Maybe not as part of this PR but it might make sense to use generics to represent a list of results, that way we don't have to create a new Result class for every list endpoint we want to support in the example.

Reply:
Yes, Agree. I have that in mind when I add new functionality.